examples/tree_1d_dgsem/elixir_euler_convergence_implicit_sparse_jacobian.jl

```diff
@@ -0,0 +1,151 @@
+using Trixi
```

Comment:
This is actually a duplicate of the elixir in the structured directory, you can delete this and the test